### DISCUSSION TITLE
Forecast Discussion

### DISCUSSION THREAD

**Discussion by mooremedia** [weather-gov/api#340] [Q&A]
Does the API support a GET for an area forecast discussion? 

Example:
https://www.wrh.noaa.gov/total_forecast/getprod.php?afos=afdlox&wfo=lox&new

If not, are there any plans to add this to the API?

Thanks so much

**Comment by StephenClouse** [comment]
All text products are available via the API, although the mapping could use a bit of work.

What you want can be found via https://api.weather.gov/products/types/AFD/locations/LOX
